### PR TITLE
Deploy to heroku

### LINKED ISSUE #20: Automate deployment for site
Deployment should be fully automated in order to achieve continuous deployment.

**Definition of done**:

- [ ] Automatic deployment when merging to `master` #39
- [x] Temporary deployment server until merging data, beta and alpha #38

### PR DESCRIPTION
**Pull request by maukoquiroga**
Fixes #20 

**Definition of done:**

- [x] Deploy to https://transport-beta.herokuapp.com/
- [x] Ensure everybody has access to app in Heroku
- [x] Add OAuth2 callback url in data.gouv.fr

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "watch": "webpack --stdin --progress --color",
     "build": "webpack",
-    "deploy": "webpack -p",
+    "deploy": "webpack -p -d",
```
Comment (l-vincent-l) [inline client/package.json]: We want to deploy the debug mode with devtools and sourcemap ?

Comment (maukoquiroga) [reply client/package.json]: No, sorry.

Comment (maukoquiroga) [reply client/package.json]: Done.

**Comment by maukoquiroga** [comment]
Rebase

**Comment by maukoquiroga** [comment]
Rebase

**Pull request merged**